pd-control.md

```diff
@@ -349,6 +349,123 @@ Usage:
     config set flow-round-by-digit 4
     ```
 
+#### `config [show | set service-middleware <option> [<key> <value> | <label> <qps|concurrency> <value>]]`
+
+`service-middleware` is a configuration module in PD, mainly used to manage and control middleware functions of PD services, such as audit logs, request rate limiting, and concurrency limiting. Starting from v8.5.0, PD supports modifying the following configurations of `service-middleware` via `pd-ctl`:
+
+- `audit`: Controls whether to enable audit logs for HTTP requests processed by PD (enabled by default). When enabled, `service-middleware` logs relevant information about HTTP requests in the PD logs.
+- `rate-limit`: Used to limit the maximum rate and concurrency of PD processing HTTP API requests.
+- `grpc-rate-limit`: Used to limit the maximum rate and concurrency of PD processing gRPC API requests.
+
+> **Note:**
+>
+> To avoid the impact of request rate limiting and concurrency limiting on PD performance, it is not recommended to modify the configurations in `service-middleware`.
```

Comment:
```suggestion
> To avoid the impact of request rate limiting and concurrency limiting on PD performance, it is not recommended to modify configurations in `service-middleware`.
```